hcert_spec.md

```diff
@@ -86,48 +89,57 @@ The Claim Key 1 is used to identify this claim.
 
 If the transfer of the EHC from the Issuer to the holder is based on a presentation-only interface (e.g., SMS, e-mail), the Raw transport encoding is obviously not applicable.
 
 ### Barcode
 
-To lower size and to improve speed and reliability in the reading process of the EHC, the CWT SHALL be compressed using ZLIB ([RFC 1950](https://tools.ietf.org/html/rfc1950)) and the Deflate compression mechanism in the format defined in ([RFC 1951](https://tools.ietf.org/html/rfc1951)). In order to better handle legacy equipment designed to operate on ASCII payloads, the compressed CWT is encoded as ASCII using [Base45](https://datatracker.ietf.org/doc/draft-faltstrom-base45) before encoded into a barcode.
+To lower size and to improve speed and reliability in the reading process of the EHC, the CWT SHALL be compressed using ZLIB ([RFC 1950](https://tools.ietf.org/html/rfc1950)) and the Deflate compression mechanism in the format defined in ([RFC 1951](https://tools.ietf.org/html/rfc1951)). 
 
-Two barcode formats are supported; AZTEC (preferred) and QR (secondary). The optical code is RECOMMENDED to be rendered on the presentation media with a diagonal size between 35 mm and 65 mm.
+Verifiers MUST check of the presence of a valid ZLIB/Deflate header (0x78, 0xDA) - or proceed without this step when absent.
+
+In order to better handle legacy equipment designed to operate on ASCII payloads, the compressed CWT is encoded as ASCII using [Base45](https://datatracker.ietf.org/doc/draft-faltstrom-base45) before encoded into a barcode.
+
+Two barcode formats are supported; AZTEC (preferred) and QR (secondary). 
 
 In order for readers to be able to detect optical payload content type, the base45 encoded data per this specification SHALL be prefixed by the string "HC1".
 
 #### AZTEC 2D Barcode
 
 To optically represent the EHC using a compact machine-readable format the Aztec 2D Barcode (ISO/IEC 24778:2008) SHOULD be used.
 
-When generating the optical code with Aztec, an error correction rate of 23% is RECOMMENDED. 
-
+When generating the optical code with Aztec, an error correction rate of 23% is RECOMMENDED. The optical code is RECOMMENDED to be rendered on the presentation media with a diagonal size between 35 mm and 65 mm.
 
 #### QR 2D Barcode
 
-Alternatively a QR barcode may be used. An error correction rate of ‘Q’ (around 25%) recommended. 
+Alternatively a QR barcode may be used. An error correction rate of ‘Q’ (around 25%) RECOMMENDED.  The Alphanumeric (Mode 2/QR Code symbols 0010) MUST be used in conjunction with Base45. 
 
+The optical code is RECOMMENDED to be rendered on the presentation media with a diagonal size between 35 mm and 65 mm when used on an optical screen with at least 4 pixels per timing cell. 
```

Comment:
Do we need "[...] when used on an optical screen with at least 4 pixels per timing cell"? Thinking that the representation media can be both screens and papers.

FWIW we've seen all sorts of scaling problems when using low resolution thermal-transfer printers. If an optical code is scaled so that not all rows of symbols are exactly square, the reliability in reading/decoding decays significantly.

https://samtrafiken.atlassian.net/wiki/spaces/BOB/pages/28999786/Considerations+for+low-resolution+printing+of+Aztec+codes

Reply:
So the reason why I put this there was that we're not quite getting this out of our Qr test results. So I put in for Qr what I am fairly certain about. And left the paper case out - as we're getting some good result with size similar as used in Germany - which is just shy of 80mm

Reply:
Raised #6